docs/README.md

```diff
@@ -1,115 +1,188 @@
-- Network Interface Card (NIC) compatible with DPDK
-- Docker installed
+- Linux server (Intel Xeon processor recommended, ex. Sapphire Rapids).
+- Network Interface Card (NIC) compatible with DPDK (ex. Intel Ethernet Controller E810-C).
+- (recommended) Update NIC drivers and firmware, more info and latest drivers [Support](#support)
 
-### Basic Installation
+#### Steps
 
-1. **Install Dependencies**, choose between options `a)` or `b)`.
+1. **Clone the repository**
 
-    a) The `setup_build_env.sh` script located in the `scripts` folder can be used for all-in-one environment preparation. The script is designed for Debian and was tested under `Ubuntu 20.04`, `Ubuntu 22.04` and `Ubuntu 24.04` environments.
-   To use this option just run the script after cloning the repository by typing `./scripts/setup_build_env.sh`
+   ```bash
+   $ git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
+   ```
+
+2. **Navigate to the Media-Communications-Mesh directory**
+
+    ```bash
+    $ cd Media-Communications-Mesh
+    ```
+
+3. **Install Dependencies**, choose between options `a)` or `b)`.
+
+    a) Use all-in-one environment preparation script. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
+
+    To use this option run the following command:
+
+    ```bash
+    sudo ./scripts/setup_build_env.sh
+    ```
 
     b) The following method is universal and should work for most Linux OS distributions.
 
-    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation.
+    - XDP-tools with eBpf: Follow the simple guide [XDP-tools](https://github.com/xdp-project/xdp-tools.git) for installation instructions.
+    - libfabric: Follow the [libfabric from source](https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source) for installation instructions.
+    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation instructions.
+    - E810 driver: Follow the [MTL NIC setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/e810.md) for installation instructions.
     - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
     - Install required packages:
+
         - Ubuntu/Debian
         ```bash
         sudo apt-get update
-        sudo apt-get install libbsd-dev
+        sudo apt-get install libbsd-dev cmake make libibverbs-dev librdmacm-dev dracut
         ```
         - Centos stream
         ```bash
-        sudo yum install -y libbsd-devel
+        sudo yum install -y libbsd-devel cmake make libibverbs-devel librdmacm-devel dracut
         ```
+
     - Install the irdma driver and libfabric
+
     ```bash
     ./scripts/setup_rdma_env.sh install
     ```
+
```

Comment:
do we need reboot? It is also in the script "./scripts/setup_build_env.sh" ?

(If not, then a) and b) are not equivalent ;)